wiki/Beatmap/Song_compilation/de.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - song compilations
+---
+
+# Song-Compilation
+
+Eine **Song-Compilation** ist eine Zusammenstellung von drei oder mehr Songs (komplett oder teilweise), die normalerweise ein Genre, einen Künstler oder etwas anderes gemeinsam haben.
+
+[Beatmaps](/wiki/Beatmap) mit Song-Compilations verwenden eine einzelne Audio-Datei, die die gesamte Liedsammlung umfasst, wobei in der Regel nur ein Refrain oder eine Strophe jedes Lieds enthalten ist. Das ist nicht zu verwechseln mit einer Beatmap, die mehrere Song-Dateien im Ordner beinhaltet.
```

Comment:
It might be colloquial, but I think it does read better. Just one more thing.
```suggestion
[Beatmaps](/wiki/Beatmap) von Song-Compilations verwenden eine einzelne Audio-Datei, die die gesamte Liedsammlung umfasst, wobei in der Regel nur ein Refrain oder eine Strophe von jedem Lied enthalten ist. Das ist nicht zu verwechseln mit einer Beatmap, die mehrere Song-Dateien im Ordner beinhaltet.
```